galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/src/main/java/dev/galasa/docker/DockerContainer.java

```diff
@@ -70,4 +70,13 @@
      */
     public String dockerEngineTag() default "PRIMARY";
 
+    /**
+     * The <code>volumes</code> will be used to indicate that a volume is to be mounted to the container when the continer is creared. The value of this
+     * field can be a comma seperated list if multiple volumes are to be mounted. The general format of the value should look like 
+     * <code>volumes = "volume_name_a:volume_mount_path, volume_name_b:volume_mount_path"</code>. This will search the docker engine for a volume that 
+     * has this name, and create one if it doesn't.
+     * @return
+     */
+    public String volumes();
```

Comment:
Should be an array of Strings at the very least.
Actually,   the volumes should be an array of DockerVolume annotations. with the volumename and the path as invidual strings.
You will need to indicate that the volume will be prefixed with the container/runname to ensure the volume names do not clash with other test runs

Reply:
prefixed with GALASA_runname_,  leave the containername off of it incase the volume is shared across containers.
GALASA being the same prefix as the containers